### DISCUSSION TITLE
如何确认filecoin 某个高度或者区块是不可逆的

### DISCUSSION THREAD

**Discussion by coder-chenda** [filecoin-project/community-china#116] [应用开发]
换句话说怎么判断某个高度或区块是已经不能被改变的

**Comment by Reskorey** [comment]
官方文档上说的是120个块确认

**Reply by Reskorey** [reply]
https://docs.filecoin.io/about-filecoin/network-performance/#financial-transfers

**Comment by dcshy** [comment]
自有技术团队近百人 机房密封日产能2p+  独立节点集群方案 可加vx：IPFS-Shy
